sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/SubscribeTransform.java

```diff
@@ -44,8 +44,6 @@
 import org.joda.time.Duration;
 
 class SubscribeTransform extends PTransform<PBegin, PCollection<SequencedMessage>> {
-  private static final Duration MAX_SLEEP_TIME = Duration.standardMinutes(1);
```

Comment:
This change comes from https://github.com/googleapis/java-pubsublite/pull/552/files

CC: @dpcollins-google